packages/noports_core/lib/src/sshnp/sshnp_params/sshnp_arg.dart

```diff
@@ -14,6 +14,21 @@ enum ArgType {
   integer,
 }
 
+enum ParseWhen {
+  normal,
+  commandLine,
+  configFile,
+
+  always,
+  never,
+}
+
+enum ParserType {
+  normal,
+  commandLine,
+  configFile,
+}
+
```

Comment:
Added ParseWhen enum and ParserType to give better (and clearer) control over what the parser should parse / ignore